packages/core/src/components/breadcrumbs/_breadcrumbs.scss

```diff
@@ -1,7 +1,7 @@
 // Copyright 2016 Palantir Technologies, Inc. All rights reserved.
 // Licensed under the terms of the LICENSE file distributed with this project.
 
-@import "../../common/icons";
+@import "../../../../icons/src/icons";
```

Comment:
this should come from `~@blueprintjs/icons/src/icons`. same with every other instance in this PR.

Reply:
but actually, maybe we don't need this import anymore? what's it for?

Reply:
There's a couple places in here where it uses variables from it for icons.